### ISSUE TITLE
The Requirement Status for Shall ID 110 is incorrect

### ISSUE THREAD

**Issue by joshuaehill** [usnistgov/90B-Shall-Statements#23]
Shall ID 110 states:
> For Section 3.1.5 and its subsections, `n_w` shall not be claimed to be greater than `n_in`. The narrowest width for non-vetted conditioning components shall be established by analysis of their designs.

The current Requirement Status is
> Required only for non-vetted conditioning components.

This is incorrect. It is possible to pass a vetted conditioning component less data than the nominal algorithm narrowest width, and in this circumstance the narrowest width is the length of the input. Please see SP 800-90B Appendix E for justification.

**Comment by joshuaehill** [comment]
For Shall ID 111, the correct value to be used is just the minimum of the input length and the stated narrowest width. This case shouldn't require separate justification.

**Comment by celic** [comment]
Yup, notation should be consistent. Tried to grab any I could find. We are using underscore `n_w` because subscripts don't show up as well on the spreadsheet.

**Comment by joshuaehill** [comment]
My editorial point is that the 90B document doesn't use a subscript here.
